decidim-core/app/helpers/decidim/filters_helper.rb

```diff
@@ -29,6 +29,14 @@ def filter_form_for(filter, url = url_for, html_options = {})
       end
     end
 
+    def filter_cache_hash(filter)
+      hash = []
+      hash << "decidim/proposals/filters"
+      hash << Digest::MD5.hexdigest(filter.to_json)
+
+      hash.join("/")
```

Comment:
Current locale must also be taken into account

Reply:
Done!